examples/3d/bloom_3d.rs

```diff
@@ -218,7 +221,7 @@ fn update_bloom_settings(
             *text = "Bloom: Off (Toggle: Space)".to_string();
 
             if keycode.just_pressed(KeyCode::Space) {
-                commands.entity(entity).insert(BloomSettings::default());
+                commands.entity(entity).insert(BloomSettings::NATURAL);
```

Comment:
nit: why change this?

Reply:
I wanted to be explicit about which default is being used. In the future, I plan on writing a PR that allows you to switch between `NATURAL`, `OLD_SCHOOL`, `SCREEN_BLUR`, etc. Explicitly saying `NATURAL` gives me room to expand it in the future.